CHANGELOG.rst

```diff
@@ -6,6 +6,86 @@
 Synapse Changelog
 *****************
 
+v2.177.0 - 2024-08-01
+=====================
+
+Automatic Migrations
+--------------------
+- Migrate Axon metrics from hive to hotcounts. Migrate Cryotank names storage
+  from hive to SafeKeyVal storage. Migrate Cortex configuration data from hive
+  to SafeKeyVal storage. Migrate Cell info and auth configuration from hive to
+  SafeKeyVal storage.
+  (`#3698 <https://github.com/vertexproject/synapse/pull/3698>`_)
+  (`#3825 <https://github.com/vertexproject/synapse/pull/3825>`_)
+- See :ref:`datamigration` for more information about automatic migrations.
+
+Model Changes
+-------------
+- Add model elements to represent the DriveSerialNumber and MachineID
+  properties of an LNK file.
+  (`#3817 <https://github.com/vertexproject/synapse/pull/3817>`_)
+- Add ``biz:deal:id`` property to track deal identifiers.
+  (`#3832 <https://github.com/vertexproject/synapse/pull/3832>`_)
+- Add ``inet:service:message:type`` property to capture message types.
+  (`#3832 <https://github.com/vertexproject/synapse/pull/3832>`_)
+- Added ``meta:rule:type`` taxonomy.
+  (`#3834 <https://github.com/vertexproject/synapse/pull/3834>`_)
+- See :ref:`userguide_model_v2_177_0` for more detailed model changes.
+
+Features and Enhancements
+-------------------------
+- Add ``synapse.lib.auth`` for non-hive based authentication.
```

Comment:
This one is an example of something that feels both internal and irrelevant to a user